src/RegistryCoordinator.sol

```diff
@@ -785,16 +785,22 @@ contract RegistryCoordinator is
         uint256 index
     ) external view returns (uint192) {
         QuorumBitmapUpdate memory quorumBitmapUpdate = _operatorBitmapHistory[operatorId][index];
+        
+        /**
+         * Validate that the update is valid for the given blockNumber:
+         * - blockNumber should be >= the update block number
+         * - the next update block number should be either 0 or strictly greater than blockNumber
+         */
+        // TODO - should this fail if quorumBitmapUpdate.quorumBitmap == 0? This will be the case for the first entry in each quorum
```

Comment:
does this todo need to be here?

Reply:
yeah i added one for each place we query in this manner

wanted the reviewers to chat about it and make sure we're happy with the status quo before i go removing each one